### DISCUSSION TITLE
Support for Polars

### DISCUSSION THREAD

**Discussion by ahmedanadwc** [sassoftware/saspy#619] [Ideas]
Hi,
Are there any plans to add support for [Polars](https://pola.rs/), as an alternative/addition to Pandas?
One way that could ease that support is using [Narwhals](https://narwhals-dev.github.io/narwhals/), at least that's what Altair used to provide [support to additional DataFrame libraries beside Pandas](https://pola.rs/posts/lightweight_plotting/)

Thanks

**Comment by tomweber-sas** [comment]
There was an issue opened about his (#610). I don't currently have the amount of time it would take to research and prototype this, is the problem.
